data/operators/power/line.json

```diff
@@ -369,18 +369,6 @@
         "power": "line"
       }
     },
-    {
-      "displayName": "Badenwerk AG",
```

Comment:
`badenwerk ag` and `badenwerk` should either be excluded or included in a matchNames of another entry

Reply:
I excluded them, as the new operator could be both either Netze BW or TransnetBW.
Badenwerk is the predecessor company of EnBW.